code/game/objects/structures/crates_lockers/closets/secure/bsdroppod.dm

```diff
@@ -1,87 +0,0 @@
-//The "BDPtarget" temp visual is created by the expressconsole, which in turn makes two things: a falling droppod animation, and the droppod itself.

```

Comment:
Revert the line ending changes

Reply:
line ending change? i repathed the name (even though it wasnt strictly necessary, i guess). @Cyberboss